app/src/main/java/is/xyz/mpv/MPVActivity.kt

```diff
@@ -365,20 +371,33 @@ class MPVActivity : Activity(), EventObserver, TouchGesturesObserver {
         return true
     }
 
-    override fun onKeyDown(keyCode: Int, event: KeyEvent): Boolean {
-        when (keyCode) {
+    private fun interceptKeyDown(event: KeyEvent): Boolean {
+        // intercept some keys to provide functionality "native" to
+        // mpv-android even if libmpv already implements these
+        var unhandeled = 0
+
+        when (event.unicodeChar.toChar()) {
```

Comment:
Why are these two hardcoded?

Reply:
`cycleSub()` will display a track switch notification in form of a toast
letting mpv handle the key won't